features/openhab-addons-external/src/main/resources/conf/gcal.cfg

```diff
@@ -0,0 +1,10 @@
+# Credentials for Google API and Google Calendar name
+
+# Copied from Credentials page on console.developers.google.com
+#client_id=
+
+# Copied from Credentials page on console.developers.google.com
+#client_secret=
+
+# The name of the Google Calendar that GCal should use
+#calendar_name=
```

Comment:
Actually, looking at this more closely, GCal Persistence has it's own calendar_name parameter, which may dictate a separate cfg file to avoid conflicting with GCal's calendar_name parameter.  WDYT?
```
#gcal-persistence:calendar_name=
#gcal-persistence:offset=
#gcal-persistence:executescript=
```